### DISCUSSION TITLE
Unable to use GitHub copilot chat extension in code server

### DISCUSSION THREAD

**Discussion by Chandrach1** [coder/code-server#6945] [Q&A]
Hi ,
I would to use GitHub copilot chat extension in code server, as I was using latest code server but still not able to use.Kindly help me

**Comment by code-asher** [comment]
Copilot is only on the Microsoft marketplace which code-server cannot use, so the only way to install it is to download the vsix and install it manually, but I am pretty sure that would be a violation of the terms of service.

**Reply by Chandrach1** [reply]
Thanks for the update but I tried to add manually unfortunately it is not reflecting in the Vscode and other way in inside of exe, I’ve edited the package.json file from 1.92 to 1.91 as the latest code server version is 1.91 for now then the extension got broken and getting error like “right hand side of “instanceof “ is not an object  “ in chat reply If any other solution could provide appreciated.

**Reply by code-asher** [reply]
You would need to install an earlier version of Copilot that does support 1.91, or wait until code-server updates with 1.92 (which should be in the next day or two).
